shaders/src/shadowing.fs

```diff
@@ -167,16 +172,46 @@ float ShadowSample_VSM(const mediump sampler2DArray shadowMap,
  */
 
 // PCF sampling
-float shadow(const mediump sampler2DArrayShadow shadowMap,
-        const uint layer, const highp vec4 shadowPosition) {
+float shadow(const bool DIRECTIONAL,
```

Comment:
I see what you are doing but it's weird to have a function parameter in upper case.

Reply:
Yeah -- it's because it's a constant at compile time, so it's like a template parameter really. I'm open to better ideas.
One idea would be to `#define constexpr const` so we'd use that in the declaration (but the IDE might choke).

Reply:
I would just call it `directional` and leave it at at that 🤷.